package.json

```diff
@@ -30,13 +31,12 @@
     "bignumber.js": "^9.0.2",
     "classnames": "^2.3.1",
     "core-js": "^3.6.5",
+    "date-fns": "^2.28.0",
     "env-cmd": "^10.1.0",
     "ethers": "^5.6.0",
     "graphql": "^15.7.2",
     "graphql-ws": "^5.6.3",
-    "lodash.debounce": "^4.0.8",
-    "lodash.memoize": "^4.1.2",
-    "lodash.once": "^4.1.1",
+    "lodash": "^4.17.21",
```

Comment:
follow up (non-blocking): now we are importing the whole of lodash we should add that linting rule that prevents developers from importing the whole thing

Reply:
#115